nutkit/protocol/requests.py

```diff
@@ -50,6 +51,8 @@ def __init__(self, uri, authToken, userAgent=None, resolverRegistered=False, dom
         self.resolverRegistered = resolverRegistered
         self.domainNameResolverRegistered = domainNameResolverRegistered
         self.connectionTimeoutMs = connectionTimeoutMs
+        if fetchSize is not None:
+            self.fetchSize = fetchSize
```

Comment:
This would, as far as I know, be the first request message that does have an optional field. I would like the condition to be removed and the field to always be part of the `NewDriver` message, as soon as all drivers support it.  
Maybe something like this:
```suggestion
        # TODO: remove assertion and condition as soon as all drivers support
        #       driver-scoped fetch-size config
        from .feature import Feature
        assert hasattr(Feature, "TMP_DRIVER_FETCH_SIZE")
        if fetchSize is not None:
            self.fetchSize = fetchSize
```